### ISSUE TITLE
Service Fabric SDK installing on ARM64 Processor

### ISSUE THREAD

**Issue by cmcfie** [microsoft/service-fabric#1416]
The issue is when you try to install the Service Fabric Runtime for Windows (specifically M1 Macs), you get an error when trying to install the Leaslayr driver as it only supports Win32/Win64 or Linux for x86 or ARM.

We would like the ability to do development for Service Fabric Applications on ARM64 based PCs, including M1 Macs and the new Windows Dev Kit 2023's, using Visual Studio, including local testing.

I have tried all kinds of options to try to get the runtime to work, but it all comes down to the fact that the LeasLayr driver doesn't support ARM64.

The issue is specifically related to the Service Fabric SDK installation and running.

**Expected behavior**
I expect that on ARM64 based computers, of which there are many now, including Macbook Pro M1's and Windows Dev Kit 2023's, that the Service Fabric SDK should be supported. Especially as Microsoft is touting that the Windows Dev Kit 2023's are the way of the future for development PCs.

**Service Fabric Runtime Version:** 
Any version.

**Environment:**
 - Standalone
 - OS: Windows 10/11

---
Assignees: /cc @microsoft/service-fabric-triage

**Comment by rustedwizard** [comment]
+1

It would be great if I could do develop on my surface pro X as I did on regular AMD64 machine. Please consider support this scenario.

**Comment by wojtekf6** [comment]
any update?

**Comment by broosted** [comment]
yup its been a year since this question was open and now snapdragon laptops are common any update ? on ARM laptops

**Comment by thejvdagaang** [comment]
does microsoft have any plans to build an ARM64 version of service fabric? it's has been a long time since this issue has been raised

**Comment by vaihoff** [comment]
Wondering the same. 
Was forced to go back to a old Intel Macbook because of this because the Service Fabric runtime does not work on the new M-chip Macbooks.

**Comment by ShikeChen-MS** [comment]
Is there any plan on this issue?

**Comment by georgedobrin** [comment]
+1
Hopefully this gets proper support now, especially because of the Surface Copilot devices

**Comment by ShikeChen-MS** [comment]
+1, given more and more devices comes with ARM64 based CPUs, this would unblock dev workflows on these devices.
